openfe/protocols/openmm_afe/equil_solvation_afe_method.py

```diff
@@ -517,6 +518,58 @@ def _validate_alchemical_components(
                           "are not currently supported")
                 raise ValueError(errmsg)
 
+    @staticmethod
+    def _validate_lambda_schedule(
+            settings: AbsoluteSolvationSettings
+    ) -> None:
+        """
+        Checks that the lambda schedule is set up correctly.
+
+        Parameters
+        ----------
+        settings : AbsoluteSolvationSettings
+          Settings object.
+
+        Raises
+        ------
+        ValueError
```

Comment:
We should also check that there are any non-zero restraints and raise a logger warning if there are (i.e. this protocol doesn't apply restraints, restraints won't be applied).